packages/pigment-css-unplugin/src/index.ts

```diff
@@ -347,7 +347,10 @@ export const plugin = createUnplugin<PigmentOptions, true>((options) => {
               if (id.endsWith('styles.css')) {
                 return theme ? generateTokenCss(theme) : _code;
               }
-              if (id.includes('pigment-css-react/theme')) {
+              if (
+                id.includes('pigment-css-react/theme') ||
+                id === `${process.env.RUNTIME_PACKAGE_NAME}/theme`
```

Comment:
```suggestion
                id.includes(`${process.env.RUNTIME_PACKAGE_NAME}/theme`)
```
Using `===` will not work from the log I tested in my local project. Here is the log of the `id`:

```
id /Users/siriwatknp/Personal-Repos/material-ui/node_modules/.pnpm/@pigment-css+react@0.0.12_@types+react@18.2.55_react@18.2.0/node_modules/@pigment-css/react/theme/index.mjs
```